vendor/modules.txt

```diff
@@ -838,4 +835,5 @@ sigs.k8s.io/structured-merge-diff/v4/value
 ## explicit; go 1.12
 sigs.k8s.io/yaml
 # github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible
+# github.com/Percona-Lab/percona-version-service/api => /home/ege/src/github.com/Percona-Lab/percona-version-service/api
 # github.com/dgrijalva/jwt-go v3.2.0+incompatible => github.com/golang-jwt/jwt/v4 v4.2.0
```

Comment:
does it just for testing?

Reply:
oops, https://github.com/percona/percona-server-mongodb-operator/pull/1010/commits/81f8bb00fa16ccaa8be8a52d7fbedb0d458e39f6